.github/workflows/release.yml

```diff
@@ -176,17 +156,41 @@ jobs:
           path: |
             dist/*.rpm
             !dist/*PAYG*.rpm
-  api-tests:
+  test-controller:
     needs: goreleaser
     runs-on: ubuntu-latest
+    container: tykio/gromit:v1.6
+    outputs:
+      conf: ${{ steps.params.outputs.conf }}
+      db: ${{ steps.params.outputs.db }}
+      pump: ${{ steps.params.outputs.pump }}
+      sink: ${{ steps.params.outputs.sink }}
+      gd_tag: ${{ steps.params.outputs.gd_tag }}
+      versions: ${{ steps.params.outputs.versions }}
+    steps:
+      - name: set params
+        id: params
+        env:
+          REPO: ${{ github.repository }}
+          # Cover pull_request_target too
+          BASE_REF: ${{startsWith(github.event_name, 'pull_request') && github.base_ref || github.ref}}
+          TAGS: ${{ needs.goreleaser.outputs.tags }}
+          IS_PR: ${{startsWith(github.event_name, 'pull_request') && 'yes' }}
+          IS_LTSPR: ${{(startsWith(github.event_name, 'pull_request') && endsWith(github.base_ref, '-lts')) && 'yes' }}
```

Comment:
@alephnull I think this should be just `IS_LTS` I don't see any IS_LTSPR env on controller.go

Reply:
I think IS_LTSPR makes most sense. controller.go can catch up.